Describes how to deal with cases where the query results for month-to… (#19908)
newrelic/docs-website: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx

    title="Month-to-date data usage"
  >
    This query shows the current full platform ingested data. In other words, it shows how much you'd be billed for your data for that month if you were billed right now.

>     Information on usage fees is calculated every hour and takes about 3 hours to generate.

Reviewer: ```suggestion
    Usage fee information is calculated hourly and takes approximately 3 hours to generate.
```
